offchain-modules/src/packages/db/entity/TronLock.ts

```diff
@@ -0,0 +1,89 @@
+import {
+  Entity,
+  PrimaryGeneratedColumn,
+  VersionColumn,
+  CreateDateColumn,
+  UpdateDateColumn,
+  Column,
+  Index,
+} from 'typeorm';
+
+export type TronAssetType = 'trx' | 'trc10' | 'trc20';
+
+@Entity()
+@Index(['tronLockTxHash', 'tronLockIndex'], { unique: true })
+export class TronLock {
+  @PrimaryGeneratedColumn()
+  id: number;
+
+  @Column()
+  tronLockTxHash: string;
+
+  @Column()
+  tronLockIndex: number;
+
+  @Index()
+  @Column()
+  tronSender: string;
+
+  @Column()
+  asset: string;
+
+  @Column()
+  assetType: string;
+
+  @Column()
+  amount: string;
+
+  @Column()
+  memo: string;
+
+  @Column()
+  timestamp: number;
+
+  @Column()
+  committee: string;
+
+  @VersionColumn()
+  version: number;
+
+  @CreateDateColumn()
+  createdAt: Date;
+
+  @UpdateDateColumn()
+  updatedAt: Date;
+
+  from({
```

Comment:
This function can be skipped.

We can create it like below.

The usage example:
```
  const c = await conn.manager.create(CkbMint, {
    id: 'unique-id',
    chain: 1,
    amount: '0x01',
    asset: '0x00000000000000000000',
    recipientLockscript: '0x00000000000000000001',
  })
  await conn.manager.save(c);
```